sizefashion/src/pages/SingleProductPage/SingleProductContent.tsx

```diff
@@ -0,0 +1,23 @@
+import { useProductsContext } from '../../context/products_context'
+
+export const SingleProductContent = () => {
+  const { singleProduct } = useProductsContext()
+
+  const { title, vendor, category } = {
+    ...singleProduct,
+  }
```

Comment:
@otaaviostein to avoid having to destructuring your own object, let's initialize your state with `null`, since it's better to initialize it not hoisting anything rather than a empty object. Doing this will effectively avoid your need to destructure it twice.